app/services/codaveri_async_api_service.rb

```diff
@@ -16,6 +16,7 @@ def post
     response = connection.post(
       headers: {
         'x-api-key' => ENV.fetch('CODAVERI_API_KEY', nil),
+        'x-api-version' => ENV.fetch('CODAVERI_API_VERSION', nil),
```

Comment:
Can hardcode the API version into the codebase.
Set as a static class constant, since when API version changes, we will also update the codebase, as well as the API version.

Reply:
done